docs/usage/11-provisioning-tcp-cloud-loadbalancers.adoc

```diff
@@ -0,0 +1,168 @@
+:sectnums:
+:numbered:
+:toc: macro
+:toc-title:
+:toclevels: 3
+:numbered!:
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+= Provisioning TCP Cloud Load-Balancers
+:icons: font
+
+[discrete]
+== Table of Contents
+toc::[]
+
+== Introduction
+
+`dklb` supports exposing Kubernetes services of type `LoadBalancer` via cloud load-balancers.
```

Comment:
```suggestion
EdgeLB supports exposing Kubernetes services of type `LoadBalancer` via cloud providers' load-balancing solution. Service resources may be annotated so that `dklb` instructs EdgeLB to enable this feature for the respective pools.
```